website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -60,6 +61,29 @@ For example, given the forward-to URL `https://my-custom-app.com/api/123`, the `
 - `forward-url-full-path`: `https://my-custom-app.com/api/123`
 - `forward-url-origin`: `https://my-custom-app.com`
 
+## Configuring custom claims
+
+Use the `X-Forward-To-Claims` header to make the Merchant Center API Gateway include extra public claims in the exchange token sent along with forwarded requests.
+
+The value must be a list of the claims separated by a space.
+
+The following claims are supported:
+* **permissions**: Optionally includes the list of applied permissions for the logged in user. The permission values are formatted as `can<permission-name>`, for example `canViewAvengers`.
```

Comment:
```suggestion
* **permissions**: Includes the list of applied permissions for the logged-in user. The permission values are formatted as `can<permission-name>`, for example `canViewAvengers`.
```

Reply:
Updated in this commit: de6456d4b4d264a8a32fad053694126c2b29b304